server/src/main/java/com/lunchtime/models/Person.java

```diff
@@ -0,0 +1,71 @@
+package com.lunchtime.models;
+
+import lombok.Getter;
+import lombok.Setter;
+import org.hibernate.annotations.CreationTimestamp;
+import org.hibernate.annotations.UpdateTimestamp;
+
+import javax.persistence.*;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import java.time.Instant;
+
+@Entity
+@Setter
+@Getter
+public class Person {
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    private Long id;
+
+    @NotBlank
+    @Column(name = "name")
+    private String name;
+
+    @NotBlank
+    @Email
+    @Column(name = "email")
+    private String email;
+
+    @NotBlank
+    @Column(name = "password")
+    private String password;
+
+    @NotBlank
+    @Column(name = "phone_number")
+    private String phoneNumber;
+
+    @Column(name = "avatar_url")
+    private String avatarUrl;
+
+    @Column(name = "is_deleted")
+    private Boolean isDeleted;
+
+    @Column(name = "role_id")
+    private Long roleId;
+
+    @Column(name = "created_at")
+    @CreationTimestamp
+    private Instant createdAt;
+
+    @Column(name = "created_by")
+    private Long createdBy;
+
+    @Column(name = "modify_at")
+    @UpdateTimestamp
+    private Instant modifyAt;
+
+    @Column(name = "modify_by")
+    private Long modifyBy;
+
+    public Boolean isDeleted() {
+        if (isDeleted == null) {
+            return false;
+        }
+        return isDeleted;
```

Comment:
Use ternary operator here

Reply:
No I don't like ternary operator, in my opinion it makes code less readable. I try to avoid it. But you can use it.